src/CalculatorUnitTests/RationalTest.cpp

```diff
@@ -0,0 +1,189 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "pch.h"
+#include <CppUnitTest.h>
+#include "Header Files/Rational.h"
+#include "Header Files/RationalMath.h"
+
+using namespace CalcEngine;
+using namespace CalcEngine::RationalMath;
+using namespace Microsoft::VisualStudio::CppUnitTestFramework;
+
+namespace CalculatorEngineTests
+{
+    TEST_CLASS(RationalTest)
+    {
+    public:
+        TEST_CLASS_INITIALIZE(CommonSetup)
+        {
+            ChangeConstants(10, 128);
+        }
+        TEST_METHOD(ModuloTest)
```

Comment:
@sanderl, thoughts on whether bulk tests like this are fine or if we should be breaking into smaller components?

Reply:
A lot of our tests have been grouping buckets of tests together has Rudy has done. I would probably break these up a little more so that we can get more coverage if one of the tests fails. The test stops execution once it fails, so the later tests within the test method would not execute. So I would probably make each section its own test. So there would be 1 test for verify the operands are not modified, 1 test for Modulo with integers, 1 for Modulo with zeros, etc.

Reply:
@sanderl I will modify that!